playbooks/tasks/portals-check-nginx.yml

```diff
@@ -0,0 +1,7 @@
+# Wait until nginx is no longer restarting
+- name: Check for Restarting in docker ps
+  shell: docker ps | grep nginx
+  register: docker_nginx
+  until: docker_nginx.stdout.find("Restarting") == -1
```

Comment:
you can use inspect templating

```bash
docker inspect -f '{{print .State.Restarting}}' nginx
```

Reply:
Or you can use docker filters:
`ansible.builtin.command: docker ps --filter name=nginx --filter status=running`
and check you got 2 lines - header and data line.

Reply:
> Or you can use docker filters:
> `ansible.builtin.command: docker ps --filter name=nginx --filter status=running`
> and check you got 2 lines - header and data line.
> 
> Anyway we should be aware of a possible issue while checking not `Restarting` or `running` status:
> Container might be in a `running` state like 100 ms before Nginx restarts and then move to `restarting` state in a loop.

is there a way to double check ? like have a condition where you need a success twice in a row with a delay of 1s for example